exchanges/lbank/lbank_wrapper.go

```diff
@@ -889,14 +889,14 @@ func (l *Lbank) FormatExchangeKlineInterval(in kline.Interval) string {
 
 // GetHistoricCandles returns candles between a time period for a set time interval
 func (l *Lbank) GetHistoricCandles(ctx context.Context, pair currency.Pair, a asset.Item, interval kline.Interval, start, end time.Time) (*kline.Item, error) {
-	req, err := l.GetKlineRequest(pair, a, interval, start, end)
+	req, err := l.GetKlineRequest(pair, a, interval, start, end, true)
```

Comment:
go run . gethistoriccandles --exchange="lbank" --pair="btc-usdt" --asset="spot" -r="720"
2023/04/13 09:39:41 rpc error: code = Unknown desc = candle open time is not UTC aligned when it should be 2021-04-24 00:00:00 +0000 UTC truncated at a 24h0m0s interval